.github/README.md

```diff
@@ -34,14 +34,25 @@ If you shutdown Docker and want to fire up the environment later, use `yarn env:
 
 ### Enable X-Debug
 
-To enable X-Debug in the containers, you need to set the environmental variables before starting up Docker:
-
+To enable X-Debug in the containers. You need to set the environmental variables before starting up Docker:
 * `export LOCAL_PHP_XDEBUG=true`
 * `export XDEBUG_HOST_IP=10.0.0.1`
+NOTE: change the IP `10.0.0.1` to the value of`$_SERVER[‘REMOTE_ADDR’]` when you load up the Docker-powered site.
+
+For Ubuntu users, run the command `ip a` and find the `docker0` alias. Copy the inet value.
+e.g. `inet 172.17.0.1/16`, use the value `172.17.0.1`.
+
+If you’ve brought up the containers already, run `yarn env stop` and `yarn env start` to reboot after setting the variable.
+
+- NOTE: For Ubuntu users: You need to include these variables every time you run `yarn env start` or `sudo yarn env start`.
+e.g `sudo LOCAL_PHP_XDEBUG=true XDEBUG_HOST_IP=172.17.0.1 yarn env start`.
+
+When you do your Path Mappings in your IDE, make sure to name your server `localhost` to do CLI debugging (including PHPUnit via `yarn test:php`).
 
-Note: change the IP `10.0.0.1` to the value of `$_SERVER['REMOTE_ADDR']` when you load up the Docker-powered site.
+- NOTE: For PHPStorm users, set the server (Settings -> Languages & Frameworks -> PHP -> Server) path mapping to the following:
```

Comment:
I think we can remove the PHPStorm specific note and just include the path mappings on its own.